porting/installing-16-04.rst

```diff
@@ -48,14 +48,14 @@ The password will be the one that you set while running halium-install.
 Common Problems
 ---------------
 
-If you have any errors while performing these steps, check see if any of the following suggestions match what you are seeing. If you have installed successfully, skip down to `Continue on`_.
+If you have any errors while performing these steps, check see if any of the following suggestions match what you are seeing. If you have installed successfully, skip down to `The Next Steps`_.
```

Comment:
check to see

Reply:
Unclear. Don't understand what is required here?

Reply:
Yes! Strange. I thought I had corrected that. It was in the original version. Fixing it now...